api/v1alpha1/aerospikecluster_validating_webhook.go

```diff
@@ -675,10 +675,16 @@ func validateAerospikeConfigUpdate(aslog logr.Logger, newConfSpec, oldConfSpec *
 	// auth-enabled and auth-disabled node can co-exist
 	oldSec, ok1 := oldConf["security"]
 	newSec, ok2 := newConf["security"]
-	if ok1 != ok2 ||
```

Comment:
Was it failing?

Reply:
Yes - because in 5.6 there is more then just  `enable-security` flag, so we must change the following: `!reflect.DeepEqual(oldSec, newSec)`